src/SFA.DAS.ApprenticeCommitments.Jobs/Apprenticeship.cs

```diff
@@ -5,6 +5,7 @@ public class Apprenticeship
         public long ApprenticeshipId { get; set; }
         public long EmployerAccountId { get; set; }
         public string Email { get; set; }
-        public string Organisation { get; set; }
+        public long AccountLegalEntityId { get; set; }
```

Comment:
At the risk of adding even more nouns, is it worth being explicit that this is for the Employer?  

I'm thinking that whilst this may be common knowledge inside EAS, it isn't quite so obvious in Apprentice Commitments.  It also matches the prefix in `EmployerAccountId` above.

```suggestion
        public long EmployerAccountLegalEntityId { get; set; }
```

Reply:
yes done